transports/msmq/routing_content_core_6.partial.md

```diff
@@ -2,9 +2,13 @@ Mapping the logical destination to the physical address containing the queue and
 
 The preferred way of configuring the physical routing is via the instance mapping file.
 
-
 ## Instance mapping file
 
+NOTE: When using instance mapping: 
+* The settings will have no effect on **audit and error queues**.
```

Comment:
does the Note-section allow for bullet points?

Reply:
I'm not sure...

Reply:
@timbussmann If you do it like this https://github.com/Particular/docs.particular.net#multi-line

Reply:
Updated to make the note multi-line.

Reply:
thanks @DavidBoike and @mauroservienti ❤️